R/call_upload_user_files.R

```diff
@@ -70,23 +131,31 @@ call_upload_user_files <- function(
   resource_filters = NA,
   resource_weight = NA,
   geo = "city",
-  acs_data_year = "2021"
-  ) {
-
-  #Define api URL
-  api_url <- stringr::str_glue("{base_url}/api/v1/upload-user-file/")
-
-  #Data Input Type Checks:
-  stopifnot(is.character(c(resource_file_path,
-                           resource_lat_column,
-                           resource_lon_column,
-                           geo,
-                           acs_data_year)
-                         )
+  acs_data_year = getOption("sedtR.year", 2021),
```

Comment:
Is the logic of this code to allow a user to set a global year option using sedtR.year (that would be different from 2021) so that they would, for example, not have to keep entering 2019 as the year argument if they wanted to use 2019 data for all of their function calls in a given session?

Reply:
Exactly. The main advantage would be for a user who is making multiple calls to the API and wants to set a default year at the outset.